src/main/java/macrobase/analysis/transform/BatchScoreFeatureTransform.java

```diff
@@ -13,23 +13,35 @@
     protected BatchTrainScore batchTrainScore;
     protected MacroBaseConf conf;
 
+    protected boolean requiresTraining;
```

Comment:
In BatchScoreFeatureTransform.java, train() and score() are called in the transform() function. I am not sure how to skip train() when calling transform() without providing an extra parameter. 

The other way I can think of is that we can add another class BatchScoreFeatureTransformWithoutTraining.java that implements transform() without train(). Would you prefer this?